source/operations/data-recovery/recover-after-site-failure.rst

```diff
@@ -1,13 +1,76 @@
 .. _minio-restore-hardware-failure-site:
 
-==========================
-Recover after Site Failure
-==========================
+=====================
+Site Failure Recovery
+=====================
 
 .. default-domain:: minio
 
 .. contents:: Table of Contents
    :local:
    :depth: 1
 
-ToDo- site replication recovery procedure
+MinIO can make the loss of an entire site, while significant, a relatively minor incident.
+Site recovery depends on the backup option you use for the site.
+
+Site Replication
+----------------
+
+:ref:`Site replication <minio-site-replication-overview>` keeps two or more MinIO deployments in sync.
+If you suffer complete site loss with site replication enabled, you can restore IAM policies, objects, metadata, and other :ref:`replicable data <minio-site-replication-what-replicates>` from the remaining site(s).
+
+1. Deploy a new MinIO site using the same ``root`` credentials as used on other deployments in the site replication configuration
```

Comment:
We could mention here that the new site can use the original hardware, but will require wiping all data (I assume). Maybe @poornas can comment on what would be required for something like a peer site having partial data loss (e.g. a span of drives failed simultaneously and were repaired, so only X% of data is actually lost).

Reply:
👍 Added.